website/content/docs/enterprise/license/faq.mdx

```diff
@@ -61,13 +68,14 @@ As Consul Enterprise approaches the expiration date, warnings will be issued in
 
 ## Q: Does this affect client agents?
 
-For existing clusters, if the clients agents are using ACLs and have a valid token, then they will be able to retrieve the license from the server.
+For existing clusters, if the clients agents are using ACLs and have a valid token, then they will be able to retrieve the license from the server if the upgrade requirements have been met. Please see the [upgrade requirements](faq#q-what-are-the-upgrade-requirements).
+
```

Comment:
The upgrade requirements state that the license needs to be present on the host, yet this section states that the clients will be able to retrieve the license from the servers if they have a valid ACL token.

This is somewhat confusing, and I'm not sure how to re-word it.

Reply:
I simplified the sentence and instead relying on the previous information provided at the top to avoid confusion.

```md
There are upgrade requirements that affect Consul Enterprise clients.
Please review the [upgrade requirements](faq#q-what-are-the-upgrade-requirements) documentation.
```